### PR TITLE
test: add tests of button component

### PR DESCRIPTION
**Pull request by Alireza-ce**
I think we should write test for our components based on these scenarios:
* Default Behavior
* Property and Attribute Changes
* User Interactions
* Conditional Rendering
* Slot Content
* Events
* Accessibility

### CODE REVIEW

**Comment by Alireza-ce** [comment]
close #120

File: src/base-button/base-button.test.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { expect, defineCE, fixture } from '@open-wc/testing';
```
Comment (amir78729) [inline src/base-button/base-button.test.ts:1]: I think this file can be deleted

File: src/button/button.test.ts [hunk 2]
```diff
@@ -1,8 +1,88 @@
...
+    const el = await fixture(
+      html`<tap-button label="Test Button"></tap-button>`,
+    );
+
+    await expect(el).to.be.accessible();
+  });
+
+  // todo after add spinner, complete this test
+  // it('should conditionally renders loading state', async () => {
+  // });
```
Comment (majidsajadi) [inline src/button/button.test.ts:48]: `spinner` is added to button. you can complete this test too.

Comment (Alireza-ce) [reply src/button/button.test.ts]: Done

**Pull request merged**